hbase-replication/src/test/java/org/apache/hadoop/hbase/replication/TestZKReplicationQueueStorage.java

```diff
@@ -206,18 +206,22 @@ public void testAddRemoveLog() throws ReplicationException {
     }
   }
 
-  // For HBASE-12865
+  // For HBASE-12865, HBASE-26482
```

Comment:
Better introduce another test? I think this test is to confirm that when there is only a single queue than we will delete the server znode so the cversion of the queuesZNode will be changed.

We should add another test where even we do not delete the server znode, the cversion of the queuesZNode will still be changed.

Reply:
Method claimQueue will not remove server znode.  And we don't need to delete the server znode to test the update of cversion. We just need to test claimQueue will update cversion of root queuesZnode.
So i reuse this method for cversion update test. But add queue '2' to enhance the robustness of the test code (to prevent cversion from changing because the server znode is deleted)

Reply:
If we do not remove the server znode, why the cversion increases? Because we have created a new server znode?

Reply:
In old code, cversion will increase because we not prepare server node of server2, so in the claimQueue method, we will create dest server znode if we not prepare. (We can find that if we prepare the znode of server2, this test will failed in old code.)
In new code, cversion will increase because out trick create and delete 'cversion_XXX' znode in root queueZnode.